packages/web/components/PageHeader.tsx

```diff
@@ -90,6 +116,27 @@ const SubMenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
   );
 };
 
+const BoxedNextImage: React.FC<
```

Comment:
Don't know if this is good practice, but that's a good workaround for next/image !

Maybe we could put this in the `design-system` package instead ? Or at least in a component of its own in `web`, so it could be reusable (🔔 MobileFooter)

Reply:
My understanding is that `NextImage`'s primary function is to load the lowest resolution image possible for a given display size.

It is using [`scrset` and `sizes`](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#resolution_switching_different_sizes), but since it doesn't enlarge an image past its original resolution that isn't really affecting these images because they're so small.

I'll:
- [ ] move the `BoxedNextImage` to the design system
- [ ] change the references in the page header
- [ ] add usage in the page footer

Reply:
I didn't dig into the details of the next/image component, and maybe I'm wrong saying it's better to use it for that particular case.

What is sure, is that there shouldn't be a placeholder image like we see in the video I uploaded while they are loading. This is not the case in other parts of the app. So creating a custom image component is a good idea, where we will be able to switch either to next/image or to a custom solution solving that issue.